hw/ip/pattgen/doc/dv/index.md

```diff
@@ -68,14 +73,17 @@ The `pattgen_base_vseq` virtual sequence is extended from `cip_base_vseq` and se
 All test sequences are extended from `pattgen_base_vseq`.
 It provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
 Some of the most commonly used tasks / functions are as follows:
-* task 1:
-* task 2:
+* setup_pattgen_channel: writing configuration values to CSR registers
+* start_pattgen_channels: randomely activate data transfer in channels
+* stop_pattgen_channels: terminate data transfer in channels 
+  and check for randomely injected errors
+* control_channels: wait for bus availability and program 
+  CSR configuration values into channels
 
 #### Functional coverage
 To ensure high quality constrained random stimulus, it is necessary to develop a functional coverage model.
 The following covergroups have been developed to prove that the test intent has been adequately met:
-* cg1:
-* cg2:
+* cg1: pattgen_op_cg
```

Comment:
I think you can instead state that the covergroups are captured at the end of the testplan. See this for example:
https://docs.opentitan.org/hw/ip/rstmgr/doc/dv/#testplan (scroll to the bottom)

This is the outcome of capturing the covergroups in the testplan HJSon.

Reply:
Wrote that covergroups are captured at the end of testplan